external/Swig.cmake

```diff
@@ -19,36 +19,30 @@
 # http://numenta.org/licenses/
 # -----------------------------------------------------------------------------
 
-option(FIND_SWIG "Use preinstalled SWIG." OFF)
+set(swig_download_url
+    "${REPOSITORY_DIR}/external/common/src/swig-3.0.2.tar.gz")
+set(pcre_download_url
+    "${REPOSITORY_DIR}/external/common/src/pcre-8.37.tar.gz")
 
-if (${FIND_SWIG})
-  find_package(SWIG)
-  # Create a dummy target to depend on.
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   add_custom_target(Swig)
+  set(swig_executable
+      ${PROJECT_SOURCE_DIR}/${PLATFORM}${BITNESS}${PLATFORM_SUFFIX}/bin/swig.exe)
+  set(swig_dir ${PROJECT_SOURCE_DIR}/common/share/swig/3.0.2)
+else()
+  ExternalProject_Add(
+    Swig
+    URL ${swig_download_url}
+    UPDATE_COMMAND ""
+    CONFIGURE_COMMAND
+      mkdir -p ${EP_BASE}/Source/Swig/Tools/ &&
+      cp ${pcre_download_url} ${EP_BASE}/Build/Swig/ &&
```

Comment:
Oh yeah I meant to change that. I started with "_url" because that is how you had it set up in your APR-related CMake scripts but intended to change it to "_path". I'll send another PR.